include/ubiquity_motor/motor_hardware.h

```diff
@@ -185,6 +187,12 @@ class MotorHardware : public hardware_interface::RobotHW {
         Joint() : position(0), velocity(0), effort(0), velocity_command(0) {}
     } joints_[2];
 
+    // MessageTypes enum in class to avoid global namespace pollution
+    enum WheelJointNumbers {
```

Comment:
Like this sort of naming but think just WheelJoint is not descriptive enough maybe Ill use WheelJointLocation  because it is then even more readable.   thanks, good input.